src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -20,7 +20,7 @@ curie_map:
   oboInOwl: http://www.geneontology.org/formats/oboInOwl#
   NCBITaxon: http://purl.obolibrary.org/obo/NCBITaxon_
   skos: http://www.w3.org/2004/02/skos/core#
-  ICD10CM: https://icd.codes/icd10cm/
+  ICD10CM: http://purl.bioontology.org/ontology/ICD10CM/
```

Comment:
Just this change. The same change has also been made in [`mondo-ingest`](https://github.com/monarch-initiative/mondo-ingest/pull/42#discussion_r929322902).